sync/src/block/extension.rs

```diff
@@ -193,6 +234,14 @@ impl Extension {
                 })
                 .collect();
 
+            let chunk_requests: Vec<RequestMessage> = requests
+                .iter()
+                .filter_map(|r| match r {
+                    (_, RequestMessage::StateChunk(..)) => Some(r.1.clone()),
```

Comment:
Binding is more clear.
`(_, msg@RequestMessage::StateChunk(..)) => Some(msg.clone()),`

Reply:
Fixed.